packages/contracts-bedrock/interfaces/governance/IProposalValidator.sol

```diff
@@ -92,25 +93,34 @@ interface IProposalValidator {
         ProposalType proposalType,
         uint8 proposalTypeConfigurator,
         bytes32 attestationUid
-    ) external returns (bytes32 proposalHash);
+    ) external returns (bytes32 proposalHash_);
```

Comment:
we should keep styling consistent so we either have `_` infront or in the end for return params